hazelcast/proxy/set.py

```diff
@@ -54,6 +62,7 @@ def add_all(self, items):
         return self._invoke(request, set_add_all_codec.decode_response)
 
     def add_listener(self, include_value=False, item_added_func=None, item_removed_func=None):
+        # type: (bool, Callable[[],Callable[[Any],List[Any]]], Callable[[],Callable[[Any],List[Any]]]) -> Future[str]
```

Comment:
`item_added` and `item_removed` are functions of the type `(ItemEvent) -> None`

Reply:
Do you mean it should be like that 
# type: (bool, Callable[[ItemEvent],None], Callable[[ItemEvent],None]) -> Future[str]